### DISCUSSION TITLE
Parallel routes with tabs

### DISCUSSION THREAD

**Discussion by fellipefreiire** [vercel/next.js#67676] [Help]
### Summary

I'm trying to create a tab component inside a parallel route.
I created 2 parallel routes @list and @tabs. @list has a list of items with a search component that uses searchParams to update the list and at @tabs i have (for testing purpose) two tabs, each tab has a list that is related to an item from the @list route, for example:

@list -> list of items
@tabs -> tab1 = sub items 1 (list) from an item of the @list and tab2 = sub items 2 (list) from the same item of the @list

i can make it render properly using search params to decide what route should be rendered.

The problem: when i switch tabs, the entire page re-render, causing the @list to make a request again and update the current list.

The goal: when i switch tabs, only the @tabs should re-render with the new data, leaving the @list untouched

### Additional information

when i click an item of @list i send the id of that id to the search param and from that id i make a request on each tab to bring the list related to that item

**Comment by logemann** [comment]
you wanna provide source code too?
